urukul.py

```diff
@@ -158,21 +164,26 @@ class Status(Module):
     |-----------+-------+-------------------------------------------|
     | RF_SW     | 4     | Actual RF switch and green LED activation |
     |           |       | (including that by EEM1.SW[0:3])          |
-    | SMP_ERR   | 4     | DDS[0:3].SMP_ERR                          |
-    | PLL_LOCK  | 4     | DDS[0:3].PLL_LOCK                         |
+    | SMP_ERR   | 4     | DDS.SMP_ERR per channel                   |
+    | PLL_LOCK  | 4     | DDS.PLL_LOCK per channel                  |
     | IFC_MODE  | 4     | IFC_MODE[0:3]                             |
     | PROTO_REV | 7     | Protocol revision (see __proto_rev__)     |
-    | DUMMY     | 1     | Not used, not usable, undefined           |
+    | DROVER    | 4     | DDS.DROVER per channel                    |
+    | DUMMY     | 21    | Not used, not usable, undefined           |
     """
+
     def __init__(self, platform, n=4):
-        self.data = Record([
-            ("rf_sw", n),
-            ("smp_err", n),
-            ("pll_lock", n),
-            ("ifc_mode", 4),
-            ("proto_rev", 7),
-            ("dummy", 1)
-        ])
+        self.data = Record(
+            [
+                ("rf_sw", n),
+                ("smp_err", n),
+                ("pll_lock", n),
+                ("ifc_mode", 4),
+                ("proto_rev", 7),
+                ("drover", n),
+                ("dummy", 25),
```

Comment:
again necessary to define dummy bits?
here it's 25 bit wide, but defined as 21 in line 172